btrfs/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,17 @@
+name: Btrfs
+files:
+  - name: btrfs.yaml
+    options:
+      - template: init_config
+      - template: instances
+        options:
+          - name: excluded_devices
+            description: List of devices to exclude.
+            value:
+              example:
+                - <DEVICE_1>
+                - <DEVICE_2>
+              type: array
+              items:
+                type: string
+          - template: instances/tags
```

Comment:
Should we use `default.yaml` templates since those include global and tag configurations?

Reply:
Hmm, they also include the service tag option.  Seems like that would add some options to the existing config, is that a pattern we are following elsewhere?